### ISSUE TITLE
Bold text in messages

### ISSUE THREAD

**Issue by TomTheDragon** [EionRobb/skype4pidgin#304]
Skype normally sends roleplay actions (like *waves*) as bold text which i cant see on Pidgin.
My question is if it would be possible to maybe add an option to change the bold text back in its original form (with the *).

**Comment by TomTheDragon** [comment]
I think the simplest way to solve this problem, is by just replacing "<b raw_pre="*" raw_post="*">some text</b>" with *some text*

**Comment by TomTheDragon** [comment]
Is there really nothing that can be done about this problem?
I wonder why no-one did mention it before because its "really" annoying if you cant see any kind of difference in the text.

**Comment by lyze237** [comment]
Skype Web displays those as:
    Bold: <b raw_pre="*" raw_post="*">Your Text here</b>
    Italic: <i raw_pre="_" raw_post="_">Your Text here</i>
    Strike Through: <s raw_pre="~" raw_post="~">Your Text here</s>

I guess that's the reason we don't see them in pidgin

**Comment by EionRobb** [comment]
Is this still a problem?

**Comment by TomTheDragon** [comment]
Yes this is still a big problem, since bold text can not be seen, which cause a lot of trouble. Specially with chat role players.
